pkgs/applications/audio/quodlibet/default.nix

```diff
@@ -1,44 +1,62 @@
-{ stdenv, fetchurl, python2Packages, wrapGAppsHook, gettext, intltool, libsoup, gnome3,
-  tag ? "",
+{ stdenv, fetchurl, buildPythonApplication, wrapGAppsHook, gettext, intltool, libsoup, gnome3, gtk3, gdk_pixbuf,
```

Comment:
Do not pass in the individual items of the Python package set but instead the package set or `python`.